index.html

```diff
@@ -3022,6 +3008,49 @@ <h3>Human-Readable Metadata</h3>
 }
 </pre>
 
+
+  <p>
+    Strings on the Web [[?STRING-META]] suggests 
```

Comment:
We don't actually suggest both first-strong and inference from language tags. We actually recommend metadata falling back to first-strong. About language tags we say:

> This approach is only recommended as a workaround for situations that prevent the use of metadata.

Because you are using JSON-LD, you have metadata for the document as a whole and for individual string values (notably `title` and `description`). What you don't have is per-string direction in language maps (e.g. `titles` and `descriptions`) and you could recommend the use of the language tag for these.

I would thus tend to say:

```
Strings on the Web recommends the use of metadata to determine the <a>base 
direction</a> of string values. When metadata is not present, applications SHOULD 
use <a>first-strong detection</a>as a fallback. Language maps do not permit per-string
metadata: implementations MAY infer the base direction from the language tag (as
a better indicator than first-strong detection).
```



Note that the <a> links are found in `i18n-glossary` (you can add to your xrefs to pick up automagically)